sigle/src/pages-lib/_error.tsx

```diff
@@ -1,77 +1,218 @@
 import React from 'react';
 import { NextPageContext } from 'next';
 import NextErrorComponent from 'next/error';
-import styled from 'styled-components';
-import tw from 'twin.macro';
+import { styled } from '../stitches.config';
 import Link from 'next/link';
 import * as Sentry from '@sentry/nextjs';
-import { Button, Container } from '../components';
-import { sigleConfig } from '../config';
+import { Button, Heading, Text } from '../ui';
+import Image from 'next/image';
+import { useAuth } from '../modules/auth/AuthContext';
+import { AppHeader } from '../modules/layout/components/AppHeader';
 
-const NotFoundContainer = styled(Container)`
-  ${tw`mt-8 mb-8 flex flex-col items-center text-center md:flex-row md:text-left`};
-`;
+const NotFoundContainer = styled('div', {
+  boxSizing: 'border-box',
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  textAlign: 'center',
+  height: '90%',
+  my: 'auto',
+  mx: 'auto',
 
-const NotFoundTextContainer = styled(Container)`
-  ${tw`flex flex-col items-center md:items-start md:pl-8`};
-`;
+  '@lg': {
+    width: '90%',
+    flexDirection: 'row-reverse',
+    textAlign: 'left',
+  },
 
-const NotFoundTitle = styled.div`
-  ${tw`mb-2 text-5xl`};
-`;
+  '@xl': {
+    width: '80%',
+  },
 
-const NotFoundSubTitle = styled.div`
-  ${tw`mb-8 text-3xl`};
-`;
+  '@2xl': {
+    width: '100%',
+  },
+});
 
-const NotFoundIllu = styled.img`
-  ${tw`mt-4 mb-4`};
-  width: 300px;
-  max-width: 100%;
+const NotFoundTextContainer = styled('div', {
+  alignItems: 'center',
+  display: 'flex',
+  flexDirection: 'column',
 
-  @media (min-width: ${sigleConfig.breakpoints.md}px) {
-    width: 520px;
+  '@md': {
+    width: '501px',
+  },
+
+  '@lg': {
+    alignItems: 'start',
+    pr: '$3',
+  },
+
+  '@xl': {
+    pr: '$3',
+  },
+});
+
+const NotFoundText = styled(Text, {
+  mb: '$6',
+  maxWidth: '400px',
+});
+
+const NotFoundWrapper = styled('div', {
+  width: '300px',
+  height: '282px',
+  position: 'relative',
+  mt: '$4',
+  mb: '$4',
+
+  '@lg': {
+    minWidth: '520px',
+    height: '488px',
+  },
+
+  '@2xl': {
+    width: '737px',
+    height: '693px',
+  },
+});
+
+const ErrorWrapper = styled('div', {
+  width: '300px',
+  height: '232px',
+  maxWidth: '100%',
+  position: 'relative',
+  mt: '$4',
+  mb: '$4',
+
+  '@lg': {
+    width: '520px',
+    height: '403px',
+  },
+
+  '@2xl': {
+    width: '765px',
+    height: '593px',
+  },
+});
+
+const NotFoundIllu = ({
+  statusCode,
+  alt,
+}: {
+  statusCode: number;
+  alt: string;
+}) => {
+  if (statusCode === 404) {
+    return (
+      <Image
+        src="/static/img/404.png"
+        alt={alt}
+        priority
+        layout="fill"
+        objectPosition="center"
+      />
+    );
   }
-`;
+
+  return (
+    <Image
+      src="/static/img/5XX.png"
+      alt={alt}
+      priority
+      layout="fill"
```

Comment:
Looks like using "fill" in the next image component is creating some issues with the responsiveness on some screens. Using either the default layout or the responsive one should fix this issue :)

![Screenshot 2022-01-25 at 14 16 42](https://user-images.githubusercontent.com/5749437/150986554-14b74221-67b8-46e2-acf8-503572fea20c.png)

Reply:
I'm currently getting the following error when trying to add any other layout than "fill":

![Screenshot 2022-01-25 at 15 24 42](https://user-images.githubusercontent.com/65421744/151013939-13c94721-eede-4329-8f6a-f44c2a605dfc.png)
 
Not sure how to work around  as we can't add a responsive width/height directly on Image component - alternatively, I could use the img component but will of course lose benefits of the Image component. What are your thoughts?